circle.yml

```diff
@@ -0,0 +1,18 @@
+dependencies: 
+  cache_directories: 
+    - ~/.cache/pip
+    - ~/wheelhouse
+  post: 
+    - .circle/services.sh
```

Comment:
`database` configuration directive is suitable for this, see: https://circleci.com/docs/configuration#database

```yaml
database:
  override:
    - ...
```